Sources/SwiftKafka/KafkaConsumer.swift

```diff
@@ -81,14 +44,14 @@ public struct KafkaConsumerMessages: AsyncSequence {
 // MARK: - KafkaConsumer
 
 /// Receive messages from the Kafka cluster.
-public final class KafkaConsumer {
+public final class KafkaConsumer: Sendable, Service { // We can do that because our stored propery is protected by a lock
```

Comment:
We can remove this comment. It is not fully true because only the `stateMachine` is protected and the compiler is enforcing it here and we don't use `@unchchecked`